pkg/engine/artifacts.go

```diff
@@ -254,6 +254,18 @@ func kubernetesManifestSettingsInit(p *api.Properties) []kubernetesComponentFile
 	}
 	o := p.OrchestratorProfile
 	k := o.KubernetesConfig
+	if k.SchedulerConfig == nil {
```

Comment:
I didn't like that we were dereferencing these maps inside the func but not protecting against nil